apps/labs/posts/pydata-extensibility-vision.md

```diff
@@ -0,0 +1,404 @@
+---
+title: 'A vision for extensibility to GPU & distributed support for SciPy, scikit-learn, scikit-image and beyond'
+published: November 17, 2021
+author: ralf-gommers
+description: 'In this post, we aim to articulate that vision and suggest a path to making it concrete, focusing on three libraries at the core of the PyData ecosystem: SciPy, scikit-learn and scikit-image. '
+category: [Array API]
```

Comment:
Possibly dumb question: Does this category name need to be explicitly enclosed in quotes, since it's in an array?

Reply:
It's not required. the example given in the readme didn't have it, and if you look at #249 where the build work you can see that filtering for Developer Workflows shows the blog added in that PR.